apps/google-analytics-4/frontend/src/components/sidebar/ChartFooter.spec.tsx

```diff
@@ -0,0 +1,16 @@
+import ChartFooter from './ChartFooter';
+import { render, screen } from '@testing-library/react';
+
+describe('Chart Footer for the sidebar app', () => {
+  it('can render the slug name', () => {
+    render(<ChartFooter slugName="my-page" viewUrl="https://contentful.com" />);
+
+    expect(screen.getByText('my-page')).toBeInTheDocument();
```

Comment:
Love all of the thorough testing 💯 curious about your choice of `toBeInTheDocument` vs `toBeVisible` ? Just in terms of testing what the user exactly sees, I believe but could be incorrect, that the former asserts it exists but not that it is visible to the user?

Reply:
My understanding is that `toBeInTheDocument` tests whether an element is present in the document regardless of visibility, but it is a good suggestion to update these to `toBeVisible` to that it is asserting whether the element is actually visible to the user. I can make this update